packages/gitbook/src/components/TableOfContents/TableOfContents.tsx

```diff
@@ -24,9 +34,12 @@ export function TableOfContents(props: {
     pages: Revision['pages'];
     ancestors: Array<RevisionPageDocument | RevisionPageGroup>;
     header?: React.ReactNode;
-    withHeaderOffset: boolean;
+    headerOffset: { sections: boolean; topHeader: boolean };
 }) {
-    const { space, customization, pages, ancestors, header, context, withHeaderOffset } = props;
+    const { space, customization, pages, ancestors, header, context, headerOffset } = props;
+
+    const withHeaderOffset = headerOffset.sections || headerOffset.topHeader;
```

Comment:
I don't think we're using this anywhere right? We can probably remove this line

Reply:
It just gets used to determine some padding.